pkg/occlient/occlient_test.go

```diff
@@ -916,9 +916,9 @@ func TestSetupForSupervisor(t *testing.T) {
 					t.Errorf("created PVC spec not matching with expected values, expected: %s, got %s", createdPVC.Spec, tt.createdPVC.Spec)
 				}
 			} else if err == nil && tt.wantErr {
-				t.Errorf("test failed, expected: %s, got %s", "error", "no error")
+				t.Error("test failed, expected: error, got no error")
 			} else if err != nil && !tt.wantErr {
-				t.Errorf("test failed, expected: %s, got %s", "no error", "error")
+				t.Errorf("test failed, expected: %s, got %s", "no error", "error: "+err.Error())
```

Comment:
Something like this might be much clearer:
t.Errorf("unexptected errror: %v", err)

Reply:
@kadel I think we should also add that no error was expected, what do you think?

Reply:
yes